### ISSUE TITLE
[Suggestion] Comment hint

### ISSUE THREAD

**Issue by generrosity** [godotengine/godot-git-plugin#91]
Hello!

This likely falls more into newbie-assistance getting used to the workflow of the plugin.

Once first opened, and once items are selected to be staged, the 'Commit Changes' button is disabled until a commit comment is added.

Could I suggest the border be highlighted once there are files waiting?  I understand 'placeholder' for textfields doesn't exist.

Might I suggest having a label above the field saying "[Required] Commit Comment".  Once a commit is made, it doesn't disable the button, and looks like it is quite happy to commit with empty comments.  This might be intentional?

I'd also like to suggest if you disabled the button, you could put there the confirmation "committed x files" text and remove the label.

There is possibly some hover-text opportunities too.

Thanks for making the plugin available :)

**Comment by ChronicallySerious** [comment]
As a part of the v2.0 efforts in https://github.com/godotengine/godot/pull/53900 and https://github.com/godotengine/godot-git-plugin/pull/84, a "Commit Message" label was added with the new commit dock UI. For screenshots, you can view some comments at https://github.com/godotengine/godot/pull/53371#issuecomment-937316734

**Comment by ChronicallySerious** [comment]
Let us know if the new UI solves your concerns or if we can do something else too!

**Comment by generrosity** [comment]
I see you have been hard at work! Thanks you very much for offering to hear my input

May I offer some feedback on the UI? I don't think there is a trivial way I can see it live, but I might have some suggestions to help those less familiar with Git but who are keen to use this.

**Comment by ChronicallySerious** [comment]
Yes please @generrosity, preferably under godotengine/godot#53900
